docs/transactions/iti41.md

```diff
@@ -47,11 +177,28 @@ Some values from the DocumentEntry are checked against the document (values in m
 
 The class and type codes are from the SNOMED CT system: `2.16.840.1.113883.6.96`.
```

Comment:
the system for class and type codes is correct, but we should also state that the system for the `formatCode `is `2.16.756.5.30.1.127.3.10.10` for completion